tasks/service_update.rb

```diff
@@ -5,7 +5,7 @@
 require 'open3'
 require 'puppet'
 
-def swarm_update(image, service)
+def service_update(image, service)
```

Comment:
As mentioned on https://github.com/puppetlabs/puppetlabs-docker/pull/546, this is a backwards incompatible change that would require a major release. To avoid that please do preserve a task under the old name to print a message and execute the new task.